app/AccountancyModule/EducationModule/presenters/EducationPresenter.php

```diff
@@ -61,4 +131,35 @@ private function getCashbookId(int $skautisEducationId): CashbookId
     {
         return $this->queryBus->handle(new EducationCashbookIdQuery(new SkautisEducationId($skautisEducationId)));
     }
+
+    /** @param array<EducationTerm> $terms */
+    private function countDays(array $terms): int
+    {
+        $days = [];
+
+        foreach ($terms as $term) {
+            $date   = $term->startDate;
+            $days[] = $date;
+
+            // Could be while(true), but don't want to risk infinite loop
+            for ($i = 0; $i < 50; ++$i) {
```

Comment:
Proč class EducationTerm nevraci pocet dnu (ten si interně spocte jako date diff) a jen na tím tady uděláme array_sum?

Reply:
Protože jednotlivé termíny se můžou překrývat a to by pak ty dny byly započítané dvakrát. A není to jen teoretická možnost, termíny jsou specifické pro kurz (a ne celou VzA), takže když má dvojkurz společný víkend, tak to bude v ISu jako 2 překrývající se (nebo třeba i jen částečně se překrývající) termíny.

Ale tohle je dost hack, kdybys měl nápad, jak to udělat líp, budu rád :D

Reply:
k čemu to číslo potřebujeme? nemáme to počítat per kurz? nemá to někde napočítané už skautis @jerrysohn @manulekonecna

Reply:
O tom, že by to měl někde napočítané SkautIS nevím (jinak bych nedělal tohle), ale nemám ta API v malíku, takže to možný je.

Tohle číslo není to nejdůležitější, důležitější jsou osobodny, podle kterých se počítá výše dotace (ostatně ta by tady mohla být vypsaná taky, to je něco, s čím jako hospodář VzA pracuješ). Ale za mě to tady smysl má, protože se podle toho odvíjí některé náklady jako je jídlo pro tým. Per kurz  mi to tolik neřekne, protože ty kurzy se většinově překrývají a počet dnů pro každý kurz není informace, kterou úplně potřebuju - tam pak spíš potřebuju vědět ty osobodny, který se z toho počítají.

Reply:
ChatGPT mi poradil tohle a přijde mi to o kousek čitelnější.
<img width="677" alt="Screenshot 2023-10-08 at 20 00 12" src="https://github.com/skaut/Skautske-hospodareni/assets/1140123/d949e815-3070-41be-83a3-b725c49836c6">

Reply:
Jo, to je o něco jednodušší :)